ICSharpCode.CodeConverter/CSharp/TypeConversionAnalyzer.cs

```diff
@@ -68,6 +68,9 @@ private TypeConversionKind AnalyzeConversion(Microsoft.CodeAnalysis.VisualBasic.
 
             if (csType is null || csConvertedType is null)
             {
+                if (alwaysExplicit && vbType != vbConvertedType) {
```

Comment:
We only return implicit if "alwaysExplicit" is set...sounds slightly backwards to me. I expect it's a naming thing?

Reply:
I meant it as "always insert an explicit type conversion". I agree it's a bit confusing.